qiskit/pulse/schedule.py

```diff
@@ -104,8 +101,16 @@ def channels(self) -> Tuple[Channel]:
         return tuple(self._timeslots.keys())
 
     @property
-    def _children(self) -> Tuple[Tuple[int, ScheduleComponent], ...]:
-        return self.__children
+    def children(self) -> Tuple[Tuple[int, ScheduleComponent], ...]:
+        """Return the child``ScheduleComponent``s of this ``Schedule`` in the
+        order they were added to the schedule.
```

Comment:
In order, and not time ordered?

Reply:
I'm not sure we make any guarantee here on their time orderedness. A child that occurs before another child in children may have instructions that happen after the other child due to the absolute nature of the time representation.